File: articles/social-identity-group-life-and-moral-polarization/fortran/polarization_distance.f90
! Synthetic polarization distance example.
! Educational scaffold only.
!
! Compile:
!   gfortran fortran/polarization_distance.f90 -o outputs/tables/polarization_distance_fortran
! Run:
!   ./outputs/tables/polarization_distance_fortran

program polarization_distance
  implicit none

  real :: group_a_mean
  real :: group_b_mean
  real :: distance

  group_a_mean = 0.72
  group_b_mean = -0.38

  distance = abs(group_a_mean - group_b_mean)

  print *, "Synthetic polarization distance:", distance
end program polarization_distance

File: articles/social-identity-group-life-and-moral-polarization/rust/polarization_distance.rs
// Synthetic polarization distance example.
// Educational scaffold only.
//
// Compile:
//   rustc rust/polarization_distance.rs -o outputs/tables/polarization_distance_rust
// Run:
//   ./outputs/tables/polarization_distance_rust

fn main() {
    let group_a_mean = 0.72_f64;
    let group_b_mean = -0.38_f64;

    let distance = (group_a_mean - group_b_mean).abs();

    println!("Synthetic polarization distance: {:.3}", distance);
}
